specification/synapse/resource-manager/Microsoft.Synapse/preview/2021-06-01-preview/integrationRuntime.json

```diff
@@ -2928,6 +3008,27 @@
         "properties"
       ]
     },
+    "IntegrationRuntimeOperationStatus": {
+      "type": "object",
+      "properties": {
+        "status": {
+          "type": "string",
+          "description": "The status of the operation."
+        },
+        "name": {
+          "type": "string",
+          "description": "The operation name."
+        },
+        "properties": {
+          "type": "object",
```

Comment:
Is this property object or should it reference an existing property object?

Reply:
it is for existing response object type property.